src/components/Todo/Todo.module.css

```diff
@@ -1,49 +1,137 @@
-.wrap {
-    width: 350px;
-    height: 550px;
-    margin: auto;
-    display: flex;
-    flex-flow: column nowrap;
-    align-items: center;
-    justify-content: space-around;
+.todo {
+  position: absolute;
+  width: 700px;
+  height: 644px;
+  margin: auto;
+  background: #fff;
+  border-radius: 9px;
+  box-shadow: 0 0 3px rgba(0, 0, 0, 0.25);
+}
+
+.wrap_todo {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+}
+
+.todo_header {
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+  margin: 5%;
 }
 
 .title {
-    font-family: 'Helvetica';
+  font: normal bold 28px Roboto, sans-serif;
+  margin: 0;
 }
 
-.item {
-    color: black;
-    list-style: none;
-    margin-left: -60;
+.item_list {
+  overflow-y: auto;
+  height: 432px;
+}
+
+.item_list::-webkit-scrollbar {
+  width: 6px;
+}
+
+.item_list::-webkit-scrollbar-thumb {
+  background: #cccccc;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#cccccc" to be "#ccc" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)